BASE/src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -64,6 +69,8 @@ static TelemetryConfiguration()
                     Activity.ForceDefaultIdFormat = true;
                 }                
             });
+
+            SelfDiagnostics = SelfDiagnosticsProvider.EvaluateSelfDiagnosticsConfig<SelfDiagnosticsFileWriter>();
```

Comment:
nit: Method name could use a better naming - this is not just Evaluating something. Its building the actual "thing" and assigning..

Reply:
I'm terrible at naming things so I'm open to suggestions here.